app/models/cloud_port.rb

```diff
@@ -0,0 +1,20 @@
+class CloudPort < ActiveRecord::Base
+  include NewWithTypeStiMixin
+  include ReportableMixin
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+  belongs_to :cloud_tenant
+  belongs_to :cloud_network
+  belongs_to :cloud_subnets
+  belongs_to :device, :polymorphic => true
+
+  has_and_belongs_to_many :security_groups
+
+  has_many :floating_ips
+
+  serialize :allowed_address_pairs
+  serialize :extra_dhcp_opts
+  serialize :binding_vif_details
```

Comment:
my guess here is `virtual_interface` ... could be renamed to `:binding_virtual_interface_details` which is a little long, but there's no tax on variable name length :wink: